packages/medusa/src/services/__mocks__/new-totals.js

```diff
@@ -0,0 +1,42 @@
+export const newTotalsServiceMock = {
+  withTransaction: function () {
+    return this
+  },
+  getLineItemsTotals: jest.fn().mockImplementation(() => {
+    return Promise.resolve(new Map())
+  }),
+  getLineItemTotals: jest.fn().mockImplementation(() => {
+    return Promise.resolve({})
+  }),
+  getOrderLineItemTotals: jest.fn().mockImplementation(() => {
+    return Promise.resolve({})
+  }),
```

Comment:
Same as the first comment

Reply:
There is no more references to specific entities such as cart or order since we are using items in reality . It is not a migration of the existing but a re work.